src/app/modules/account/login/login.component.scss

```diff
@@ -21,14 +21,13 @@
 .login {
   margin: 100px 0;
 
-  .illustration img {
-    max-width: 500px;
-  }
-
-  // illustration will be invisible on smaller screens
-  @include media-breakpoint-down(lg) {
-    .illustration {
-      display: none;
```

Comment:
Why remove this? I don't think this image is going to add much value for small(er) screens. 
Also, I don't think the image resizes properly (unlike the one on the home page)

Reply:
Because it makes the page more interesting and I had room left to pace it. but I'll remove it